src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -141,6 +141,8 @@ private static bool GetLinqExpressionsBuiltWithIsInterpretingOnly()
 
         public static bool IsMetadataUpdateSupported => !IsNativeAot;
 
+        public static bool IsBoxedByRefLikeValuesSupported => !IsNativeAot;
```

Comment:
What does `IsBoxedByRefLikeValuesSupported` mean? Boxing of byref-like values should always throw exceptions. It should be unconditionally supported.

Reply:
It looks like a NativeAOT bug. Can you try to extract a small repro that fails with NativeAOT with this exception and works fine everywhere else?

Reply:
I think we should block the test on https://github.com/dotnet/runtime/issues/72548. Looks like that bug.